galera/src/certification.cpp

```diff
@@ -873,6 +901,7 @@ galera::Certification::TestResult galera::Certification::do_test_nbo(
     }
     else
     {
+        assert(is_nbo_end);
```

Comment:
Should be inside `#ifndef NDEBUG`

Reply:
assert is an empty macro if NDEBUG is defined